src/views/Entries/QualityControl/EntryCard/styles.scss

```
@@ -115,24 +116,53 @@
 
     .middle {
         flex-grow: 1;
-        padding: var(--spacing-small) var(--spacing-medium);
+        padding: var(--spacing-small) var(--spacing-large);
         overflow: auto;
 
         .excerpt-output {
             padding: 0;
             height: 240px;
         }
     }
+    .expand-button {
+        border: none !important;
+        background: var(--color-foreground-alt) !important;
```

```suggestion
        border: none !important;
        outline: none;
        background: var(--color-foreground-alt) !important;
```
a black outline appears when clicking on the button. this seems to solve the issue.

---

shouldn't !important be used sparsely ?

---

supposed to be used sparsely, yes. But we're trying to change the styles of a component from react store. Unless !important is used, the component style can't be overridden.